### PR TITLE
Adding PBS Adapter

### PR DESCRIPTION
**Pull request by avincigu**

### CODE REVIEW

File: procedures/src/main/java/com/intel/dai/procedures/InternalJobInfoJobCompleted.java [hunk 1]
```diff
@@ -50,7 +54,13 @@ public VoltTable run(String sJobId, String sWorkDir, String sWlmJobState, long l
         long lSearchForTsEndBoundInMicroSecs = lStartTsInMicroSecs + (1500 * 1000L);
 
         // Check & see if there is already a row containing the JobInfo for the specified job.
-        voltQueueSQL(selectJobInfoBetween, EXPECT_ZERO_OR_ONE_ROW, sJobId, lStartTsInMicroSecs, lSearchForTsEndBoundInMicroSecs);
+        if (lStartTsInMicroSecs == -1L) {
+            voltQueueSQL(selectJobInfoNoTime, EXPECT_ZERO_OR_ONE_ROW, sJobId);
```
Comment (paul-amonson) [inline procedures/src/main/java/com/intel/dai/procedures/InternalJobInfoJobCompleted.java:58]: So you are not using the dai_network_listener component? Your adapter has a direct dependency on voltdb? Shouldn't you make functions in the dsapi/dsimpl?

File: configurations/hw1/AdapterWlmPBS.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "bootstrap.servers": "admin:9092",
+    "group.id": "pbs_runjobs",
+    "schema.registry.url": "http://admin:8081",
+    "auto.commit.enable": "true",
+    "auto.offset.reset": "earliest",
+    "specific.avro.reader": "true",
```
Comment (paul-amonson) [inline configurations/hw1/AdapterWlmPBS.json:7]: Will the PBS pro messages be AVRO encoded JSON? Just a question.

**Review by paul-amonson — APPROVED** [review]

File: configurations/hw1/AdapterWlmPBS.json (hunk 2 above)
Comment (paul-amonson) [reply configurations/hw1/AdapterWlmPBS.json:7]: Cool.

File: resource_managers/src/main/java/com/intel/dai/resource_managers/cobalt/AdapterWlmCobalt.java [hunk 3]
```diff
@@ -110,53 +110,37 @@ public long handleInputFromExternalComponent(String[] aWiParms) throws ProviderE
...
-            catch (Exception e) {
-                // Log the exception, generate a RAS event and continue parsing the console and varlog messages
-                e.printStackTrace();
-                log_.exception(e, "handleDelivery - Exception occurred while processing an individual message - '" + message + "'!");
-                String eventtype = "RasProvException";
-                String instancedata = "AdapterName=" + adapter.getName() + ", Exception=" + e.toString();
-                raseventlog.logRasEventNoEffectedJob(eventtype, instancedata, null, System.currentTimeMillis() * 1000L, adapter.getType(), workQueue.workItemId());
+            // This record came in via cqm log.
+            else if (subject.equals("InputFromLogstashForJobData")) {
+                handleCobaltJobMessages(message, aLineCols);
```
Comment (yhuan47) [inline resource_managers/src/main/java/com/intel/dai/resource_managers/cobalt/AdapterWlmCobalt.java:129]: Noticed the file name and function name still come with Cobalt, is it supposed to changed to pbs?

Comment (avincigu) [reply resource_managers/src/main/java/com/intel/dai/resource_managers/cobalt/AdapterWlmCobalt.java:129]: This file does the Cobalt processing. For PBS you will want to look at AdpaterPBS.java

**Review by yhuan47 — APPROVED** [review]

**Pull request merged**